patches/services-network-public-mojom-cookie_manager.mojom.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/services/network/public/mojom/cookie_manager.mojom b/services/network/public/mojom/cookie_manager.mojom
+index efb5fa353efa87c39de48c477c0bddd8dc4d4b19..397c7642a767430432c0db7f035289cbab74391e 100644
+--- a/services/network/public/mojom/cookie_manager.mojom
++++ b/services/network/public/mojom/cookie_manager.mojom
+@@ -114,6 +114,7 @@ struct CookieOptions {
+   CookieSameSiteContext same_site_cookie_context;
+   bool update_access_time = true;
+   bool return_excluded_cookies = false;
++  string ephemeral_storage_domain;
+ };
+ 
+ // See net/cookies/canonical_cookie.{h,cc} for documentation.
+@@ -250,6 +251,11 @@ struct CookieDeletionFilter {
+ 
+   // Delete session/persistent cookies.
+   CookieDeletionSessionControl session_control = IGNORE_CONTROL;
++
```

Comment:
never add blank lines or comments to patches

Reply:
:+1: